app/models/learner-group.js

```diff
@@ -19,6 +19,7 @@ export default DS.Model.extend({
     }
   ),
   instructorGroups: DS.hasMany('instructor-group', {async: true}),
+  offering: DS.belongsTo('offering', {async: true, inverse: 'learnerGroups'}),
```

Comment:
Can you remind me why this is correct?  Seems wrong since a learnerGroup has many offerings and never just one.